test/plugins/test_ignore_long_lines.py

```diff
@@ -30,7 +30,7 @@ def test_ignore_long_lines():
         # .. toggle_description: this is a good toggle, it goes back and forth, toggling a lot, which is what we expect toggles to do.
     """
     msg_ids = "line-too-long"
-    messages = run_pylint(source, msg_ids)
+    messages = run_pylint(source, msg_ids, "--ignore-long-lines", r"^\s*(# )?((<?https?://\S+>?)|(\.\. \w+: .*))$")
```

Comment:
I’m confused. This doesn’t seem like it is testing our rules any more, but seems like it is testing regex capabilities now. Maybe I’m missing something?

Reply:
This test was actually testing the pylintrc file validity... IMHO we could simply remove this test, as it's not really testing our own code.

Reply:
I'm still not 100% clear.  Is it true that this used to test that a specific `pylintrc` file had expected config?

Does pylint now require `--rcfile=pylintrc`? If so, would this get us back to the original test?

I agree that the current test just seems to be test pylint, and does not verify the pylintrc file, so I don't think we have the right update here.

Did something change so that that `pylintrc` file is no longer used for the test?  If so, is there a different update that would use the previous `pytlintrc`

Reply:
Reminder to respond to this comment.  Thanks.

Reply:
To be honest, I'm not quite sure why this change is required for pylint>=2.6.0. I concluded that the pylintrc file was not loaded for the tests, but I don't know why, because I did not dive deep enough in the pylint source code.

That being said, you are right that adding `rcfile=../../pylintrc` does fix the test. So what I will do is to add this to the unit test.